tsconfig.production.json

```diff
@@ -14,11 +15,12 @@
     "noUnusedParameters": true,
     "outDir": "lib",
     "pretty": true,
+    "resolveJsonModule": true,
     "rootDir": ".",
-    "sourceMap": true,
+    "sourceMap": false,
```

Comment:
Why?

Reply:
Eslint plugins only work with `.js` files, `sourceMap ` generates `.js.map` files.

Reply:
Maps also provide a way to have better stack traces in case of errors, I don't see why would we not have them.